src/scripts/get_lkg_hash.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+CIRCLE_FETCH_PAGE_SIZE=100
+CIRCLE_FETCH_MAX_PAGES=100
+response=""
+function get_recent_builds() {
+    offset=$(($1*CIRCLE_FETCH_PAGE_SIZE))
+    url="https://circleci.com/api/v1/project/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/tree/${CIRCLE_BRANCH}?circle-token=${CIRCLE_TOKEN}&limit=100&offset=${offset}&filter=successful"
```

Comment:
Fix indentation?

Reply:
What is the issue with the indentation?

Reply:
The if body was off when I looked earlier -- looks ok now.